examples/sampling/graphbolt/node_classification.py

```diff
@@ -96,7 +70,7 @@ def create_dataloader(args, graph, features, itemset, is_train=True):
     # referred to as a 'mini-batch'. (The term 'mini-batch' is used here to
     # indicate a subset of the entire dataset that is processed together. This
     # is in contrast to processing the entire dataset, known as a 'full batch'.)
-    # 'is_train': Determining if data should be shuffled. (Shuffling is
+    # 'job': Determines whether data should be shuffled. (Shuffling is
```

Comment:
The comment should be updated to, which values are accepted, and optionally explain the difference between jobs.